portal/src/components/v2/ColorInput/ColorInput.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback, useRef, useState, useEffect } from "react";
+import { TextField as RadixTextField } from "@radix-ui/themes";
+import { TextInput } from "../TextField/TextField";
+
+import styles from "./ColorInput.module.css";
+
+export type ColorHex = string;
+
+// Note: Only the format of #xxxxxx is accepted by color input, so we do not handle other color format
+const COLOR_REGEX = /^#?[0-9a-fA-F]{6}$/;
+
+export interface ColorInputProps {
+  value: ColorHex;
+  onValueChange?: (value: ColorHex) => void;
+}
+
+export function ColorInput({
```

Comment:
Add a story for ColorInput

Reply:
Same as ImageInput, it is not a component in the storybook design.

Reply:
ColorInput should have a story.